client/app/lib/util/cleanup.coffee

```diff
@@ -3,16 +3,55 @@ kookies = require 'kookies'
 
 toBeDeleted = [
   'clientIPAddress' # an old cookie set by koding backend.
+  'kdproxy-preferred-domain' # old koding cookie
   'connect.sid'     # express cookie - infact this is session cookie, here for just in case.
-  'pnctest'         # set by PubNub for testing.
-  'hubspotutk'      # hubspot
-  'hsfirstvisit'    # hubspot
+  # 'pnctest'       # set by PubNub for testing. it is set on every load, so not necessary to remove.
   '__ssid'          # google map
+  'gsScrollPos'     # session cookie, wont be cleaned, not necessary
 
-  # 'ajs_anonymous_id' # font?
-  # 'ajs_group_id'     # font?
-  # 'ajs_user_id'      # font?
   # 'koding-teams'     # set by koding client, should be moved to session storage.
+
+  # hubspot
+  '__hstc'
+  '__hssrc'
+  '__hssc'
+  'hsPagesViewedThisSession'
+  'hubspotutk'
+  'hubspot.hub.id'
+  'hsfirstvisit'
+  'hubspotauth'
+  'hubspotauthcms'
+  'hubspotauthremember'
+  'hubspotutktzo'
+  '_hs_opt_out'
+  '__hluid'
+
+  # font?
+  'ajs_anonymous_id'
+  'ajs_group_id'
+  'ajs_user_id'
```

Comment:
nice! i wasnt sure about the source of these, came across to "set by custom font provider" in many places that is why i wrote `font?` removing immediately.